### ISSUE TITLE
Support Updated CSS nesting Syntax

### ISSUE THREAD

**Issue by wesbos** [chakra-ui/panda#2804]
### Description

Using native CSS syntax without the. `&` does not work. The spec originally needed the `&` but was removed after.

```jsx
import { css } from 'styled-system/css';

export const App = () => {
  return (
    <div className={css`
      background: red;
      p {
        background: orange; /* Doesnt work */
      }
      & p { background: blue; } /* Does work */
    `}>
      <p>Style me</p>
    </div>
  );
};

```

### Link to Reproduction

https://play.panda-css.com/AzUYVWTNqj

### Panda CSS Version

0.45.0

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by anubra266** [comment]
Hiya @wesbos I'm not sure if I understand the issue exactly.
But nested styling in panda for both literals and object require the `&`
https://panda-css.com/docs/concepts/template-literals#nested-styles

**Comment by wesbos** [comment]
Hey - thanks for the reply. The issue is that the standard CSS nesting spec does not require it. I'm actually moving over some styled components and I have several hundred that need to be edited to add the &. Is it not possible to support the standard css nesting spec?

**Comment by anubra266** [comment]
You're right @wesbos. It's css spec, so panda would support it. Twil be put on the radar.
Thanks for pointing it out
